<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lâm Thuỳ Dung</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(248, 250, 252);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Book Management</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxBook">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>291</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(246, 250, 252);
</string>
    </property>
    <property name="title">
     <string>Books:</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="font">
        <font>
         <family>#9Slide03 Arima Madurai ExtraBo</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(188, 204, 220);</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>267</width>
          <height>363</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutButtons"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxBookDetails">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>341</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(246, 250, 252);</string>
    </property>
    <property name="title">
     <string>Book Details:</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ISBN:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditISBN">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>39</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(217, 234, 253);
border-radius: 6px;
border: 0.5px solid black;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTitle">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>79</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(217, 234, 253);
border-radius: 6px;
border: 0.5px solid black;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Author:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAuthor">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>119</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(217, 234, 253);
border-radius: 6px;
border: 0.5px solid black;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Year:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditYear">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>159</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(217, 234, 253);
border-radius: 6px;
border: 0.5px solid black;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSave">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(217, 234, 253);
border-radius: 10px;
border: 0.5px solid black;
</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemove">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(217, 234, 253);
border-radius: 10px;
border: 0.5px solid black;
</string>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Publisher:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPublisher">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>199</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(217, 234, 253);
border-radius: 6px;
border: 0.5px solid black;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(246, 250, 252);</string>
    </property>
    <property name="title">
     <string>More functions:</string>
    </property>
    <widget class="QPushButton" name="pushButtonSearchTitle">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(217, 234, 253);
border-radius: 10px;
border: 0.5px solid black;
</string>
     </property>
     <property name="text">
      <string>Search Title</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSearchISBN">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(217, 234, 253);
border-radius: 10px;
border: 0.5px solid black;
</string>
     </property>
     <property name="text">
      <string>Search ISBN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFilterPublisher">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(217, 234, 253);
border-radius: 10px;
border: 0.5px solid black;
</string>
     </property>
     <property name="text">
      <string>Filter Publisher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFilterYear">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(217, 234, 253);
border-radius: 10px;
border: 0.5px solid black;
</string>
     </property>
     <property name="text">
      <string>Filter Year</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
